Week2/alaa.js

```diff
@@ -0,0 +1,97 @@
+//1
+console.log('Hallo', 'wereld!'); // Dutch
+console.log('hola', 'mundo!'); //spanish
+console.log('مرحبا', 'العالم!'); //Arabic
+//2
+console.log("I'm awesome"); // should use two quotations instead of one
```

Comment:
Just for your information, you can also use escape characters using backslash i.e `\` like: `console.log('I\'m awesome');`. But the way you did is also correct.